### PR TITLE
Bug/responsive graphs

### PR DESCRIPTION
**Pull request by Ethan-Edmond**
# Description

The graphs on the graph page went to the edges of the page on mobile. I added padding. The graphs didn't resize well, due to a bunch of size constraints. I made them take up 100% of their containers up to 1200px, allowing them to resize better. I also chopped a bunch of unused selectors from index.css, corrected the inaccurate pie graph legend and moved the graph-specific css files to less files.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [X] `npm test`

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: src/components/graphs/linegraph/LineGraph.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React, { useEffect, useState } from 'react';
 import { Line } from 'react-chartjs-2';
 import { options } from '../assets';
 
+import './LineGraph.css';
```
Comment (ahmedseragcodes) [inline src/components/graphs/linegraph/LineGraph.js]: The change above this shows that LineGraph.CSS was deleted, but then its being imported here.

Comment (ahmedseragcodes) [reply src/components/graphs/linegraph/LineGraph.js]: Nvm, misread haha, something about the file not being rendered by default

**Review by ahmedseragcodes — APPROVED** [review]
Changes make sense, don't see bugs or new-room for bugs

File: src/components/graphs/linegraph/LineGraph.js (hunk 1 above)
Comment (Ethan-Edmond) [reply src/components/graphs/linegraph/LineGraph.js]: Yeah, I already changed the import to './LineGraph.less'

File: src/components/graphs/bargraph/BarGraph.js [hunk 2]
```diff
@@ -20,23 +79,50 @@ const def = {
...
+    };
+    queryMatcher.addListener(callback);
+
+    return () => {
+      queryMatcher.removeListener(callback);
+    };
+  }, []);
 
   useEffect(() => {
-    if (incidentCount) {
```
Comment (ahmedseragcodes) [inline src/components/graphs/bargraph/BarGraph.js:30]: Do we know what happens if incidentCount is null? Seems like this was checking for that prior to creating the data object / dataset array

Comment (Ethan-Edmond) [reply src/components/graphs/bargraph/BarGraph.js:30]: It only checks if count is null, but considering we already have a default data set, this case is already covered, so I removed it.

Comment (ahmedseragcodes) [reply src/components/graphs/bargraph/BarGraph.js:30]: Ahhh ok, got it

**Review by ahmedseragcodes — APPROVED** [review]
Spoke with author regarding if statement incidentcount concerns, otherwise changes make sense, don't see any bugs, approved

File: src/components/graphs/bargraph/BarGraph.js (hunk 2 above)
Comment (Ethan-Edmond) [reply src/components/graphs/bargraph/BarGraph.js:30]: Actually, after testing it looks like GraphContainer is taking care of the case of a null count.

**Comment by Ethan-Edmond** [comment]
I've split this pr into smaller chunks since it's a bit larger than I expected, so those can be found on bug/responsive-graphs-1, 2 or 3, with each being dependent on the earlier ones.

**Pull request closed without merging**